src/vehicle_use.cpp

```diff
@@ -1092,6 +1093,38 @@ void vehicle::play_chimes()
     }
 }
 
+void vehicle::crash_terrain_around() {
```

Comment:
There is `void vehicle::transform_terrain()` function which already does something like `crash_terrain_around` does - maybe you can combine it with your code to avoid code duplication?

Reply:
unfortunately, not. I think about it, but it has a lot of differences.